options/ansi/generic/time-stubs.cpp

```diff
@@ -578,10 +579,445 @@ char *ctime_r(const time_t *clock, char *buf) {
+                case 'g':
+                    break;
+                case 'G':
+                    break;
+                case 'u': {
+                    int product = 0;
+                    int n = read_number(&product, &state->input_index, input, 1);
+                    if(n == 0)
+                        return NULL;
+                    tm->tm_wday = product - 1;
+                    break;
+                }
+                case 'V':
+                    break;
+                case 'z':
+                    break;
+                case 'Z':
+                    break;
+                case 's': //end of GNU extensions
+                    break;
+                case 'E': { //locale-dependent date & time representation
+                    state->format_index++;
+                    switch(format[state->format_index]){
+                        case 'c':
+                            break;
+                        case 'C':
+                            break;
+                        case 'x':
+                            break;
+                        case 'X':
+                            break;
+                        case 'y':
+                            break;
+                        case 'Y':
+                            break;
+                        default:
+                            return NULL;
+                    }
+                }
+                case 'O': { //locale-dependent numeric symbols
+                    state->format_index++;
+                    switch(format[state->format_index]){
+                        case 'd':
+                        case 'e':
+                            break;
+                        case 'H':
+                            break;
+                        case 'I':
+                            break;
+                        case 'm':
+                            break;
+                        case 'M':
+                            break;
+                        case 'S':
+                            break;
+                        case 'U':
+                            break;
+                        case 'w':
+                            break;
+                        case 'W':
+                            break;
+                        case 'y':
+                            break;
+                        default:
+                            return NULL;
+                    }
+                }
+                default:
+                    return NULL;
+            }
+        }
+        else if(isspace(format[state->format_index])){
+            while(isspace(input[state->input_index++]));
+            state->input_index--;
+        }
+        else {
+            if(format[state->format_index] != input[state->input_index++])
+                return NULL;
+        }
+        state->format_index++;
+    }
+
+    return (char*)input + state->input_index;
+}
+
+} //anonymous namespace
+
+/*char *strptime(const char *__restrict, const char *__restrict,
```

Comment:
Just remove this

Reply:
yeah I realized I forgot to remove that just after I shut off my PC

Reply:
This is now resolved.